pyvista/core/pointset.py

```diff
@@ -122,6 +128,175 @@ def remove_cells(self, ind, inplace=True):
+        >>> _ = mesh.points_to_double()
+        >>> mesh.points.dtype
+        dtype('float64')
+
+        """
+        if self.points.dtype != np.double:
+            self.points = self.points.astype(np.double)
+        return self
+
+    def translate(self, xyz: Union[list, tuple, np.ndarray], transform_all_input_vectors=False, inplace=None):
+        """Translate the mesh.
+
+        To Do
+        -----
+        `transform_all_input_vectors` is not handled when modifying inplace
+
+        Parameters
+        ----------
+        xyz : list or tuple or np.ndarray
+            Length 3 list, tuple or array.
+
+        Examples
+        --------
+        Create a sphere and translate it by ``(2, 1, 2)``.
+
+        >>> import pyvista
+        >>> mesh = pyvista.Sphere()
+        >>> mesh.center
+        [0.0, 0.0, 0.0]
+        >>> trans = mesh.translate((2, 1, 2), inplace=True)
+        >>> trans.center
+        [2.0, 1.0, 2.0]
+
+        """
+        if inplace is None:
+            # Raise a warning about inplace changing
+            warnings.warn(DEFAULT_INPLACE_WARNING)
+            inplace = True
+        if inplace:
+            self.points += np.asarray(xyz)  # type: ignore
+            return self
+        return super().translate(xyz, transform_all_input_vectors=transform_all_input_vectors, inplace=inplace)
+
+    def scale(self, xyz: Union[list, tuple, np.ndarray], transform_all_input_vectors=False, inplace=None):
+        """Scale the mesh.
+
+        To Do
+        -----
+        `transform_all_input_vectors` is not handled when modifying inplace
+
+        Parameters
+        ----------
+        xyz : scale factor list or tuple or np.ndarray
+            Length 3 list, tuple or array.
+
+        Examples
+        --------
+        >>> import pyvista
+        >>> from pyvista import examples
+        >>> pl = pyvista.Plotter(shape=(1, 2))
+        >>> pl.subplot(0, 0)
+        >>> pl.show_axes()
+        >>> _ = pl.show_grid()
+        >>> mesh1 = examples.download_teapot()
+        >>> _ = pl.add_mesh(mesh1)
+        >>> pl.subplot(0, 1)
+        >>> pl.show_axes()
+        >>> _ = pl.show_grid()
+        >>> mesh2 = mesh1.scale([10.0, 10.0, 10.0], inplace=False)
+        >>> _ = pl.add_mesh(mesh2)
+        >>> pl.show(cpos="xy")
+        """
+        if inplace is None:
+            # Raise a warning about inplace changing
+            warnings.warn(DEFAULT_INPLACE_WARNING)
+            inplace = True
+        if inplace:
+            self.points *= np.asarray(xyz)  # type: ignore
+            return self
+        return super().scale(xyz, transform_all_input_vectors=transform_all_input_vectors, inplace=inplace)
+
+    @wraps(DataSet.flip_x)
+    def flip_x(self, *args, **kwargs):
+        """Wrap ``DataSet.flip_x``."""
+        if kwargs.get('inplace', None) is None:
+            # Raise a warning about inplace changing
+            warnings.warn(DEFAULT_INPLACE_WARNING)
+            kwargs.setdefault('inplace', True)
```

Comment:
Here and in several other similar places:
1. `kwargs.get(..., None)` is just `kwargs.get(...)`
2. the premise inside this `if` is that `kwargs` doesn't contain `inplace` or its value is `None`, so `kwargs.setdefault` seems needless (making the code less transparent). So isn't that just `kwargs['inplace'] = True`?

Reply:
1. Good point, I'll adjust this
2. Ah good catch, my sleepy brain last night didn't realize that. I will fix